SYCL/ESIMD/api/functional/functions/functions_1d_select.hpp

```diff
@@ -0,0 +1,281 @@
+//===-- functions_1d_select.hpp - Functions for tests on simd rvalue select
+//      function. ---------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions for tests on simd 1d select function.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "../shared_element.hpp"
+#include "common.hpp"
+
+// for std::ceil
+#include <cmath>
+// for std::numeric_limits
+#include <limits>
+
+namespace esimd = sycl::ext::intel::experimental::esimd;
```

Comment:
```suggestion
namespace esimd = sycl::ext::intel::esimd;
```
Let's move it to the `functions/common.hpp` for this PR. What would you say?

Reply:
I've applied this comment in this commit: https://github.com/intel/llvm-test-suite/pull/875/commits/ee611b346fdc32de475536463be8cb3d45e3f5ea